packages/controls-navigator/__tests__/navigator.spec.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-undef */
+import { mount, enableAutoUnmount } from "@vue/test-utils";
```

Comment:
I really want the navigator is independent of inquiry GUI. So I would like keep the unit test in its own folder since we can run the test when building and we also have the own coverageThreshold to control the test coverage.